x-pack/filebeat/module/mssql/log/ingest/pipeline.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
We can use yaml now for pipelines :slightly_smiling_face: https://github.com/elastic/beats/pull/11209

Reply:
No, thank you :joy: :joy: :joy: :joy: :joy: